sass/basics.scss

```diff
@@ -37,6 +37,33 @@ a {
   color: red;
 }
 
+.tipMessage {
+  border: 1px solid #c3e0c9;
+  background-color: #f3f9f4;
+  color: #526d8a;
+  font-size: smaller;
+  padding: 0.5em;
+  margin-bottom: 0.5em;
+}
+
+.warningMessage {
+  border: 1px solid #feedbb;
+  background-color: #fffdf7;
+  color: #526d8a;
```

Comment:
Do we use this text colour elsewhere? If not, it might look more coherent if we use the existing text paragraph colour.